pkg/statistics/store_collection.go

```diff
@@ -111,6 +112,15 @@ func (s *storeStatistics) Observe(store *core.StoreInfo, stats *StoresStats) {
 	s.RegionCount += store.GetRegionCount()
 	s.LeaderCount += store.GetLeaderCount()
 	s.WitnessCount += store.GetWitnessCount()
+	limit, ok := store.GetStoreLimit().(*storelimit.SlidingWindows)
+	if ok {
+		cap := limit.GetCap()
+		storeStatusGauge.WithLabelValues(storeAddress, id, "windows_size").Set(float64(cap))
+		for i, use := range limit.GetUsed() {
+			storeStatusGauge.WithLabelValues(storeAddress, id, "windows_used_level_"+strconv.Itoa(i)).Set(float64(use))
```

Comment:
Do we need to change the levels 0, 1, 2 to low, medium, high...?

Reply:
good suggestion.